### PR TITLE
Fix some lint warning

### PR DESCRIPTION
**Pull request by emartynov**
## :loudspeaker: Type of change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [x] Refactoring

## :scroll: Description

First banch of lint fixes

## :bulb: Motivation and Context

This is a showcase app. I think it should be an example of how to keep your project in good shape. 

## :green_heart: How did you test it?
Compile and run.

## :pencil: Checklist

- [x] I ran `./gradlew spotlessApply` before submitting the PR
- [x] I reviewed submitted code

## :crystal_ball: Next steps
Continue with lint report fixes.

Partial fix for #745

### CODE REVIEW

File: core/src/main/java/io/plaidapp/core/util/CollapsingTextHelper.java [hunk 1]
```diff
@@ -478,12 +459,6 @@ private void setInterpolatedTextSize(final float textSize) {
             }
             mIsRtl = calculateIsRtl(mTextToDraw);
         }
-        // Use our texture if the scale isn't 1.0
-        mUseTexture = USE_SCALING_TEXTURE && mScale != 1f;
```
Comment (florina-muntenescu) [inline core/src/main/java/io/plaidapp/core/util/CollapsingTextHelper.java]: why was this removed?

Comment (emartynov) [reply core/src/main/java/io/plaidapp/core/util/CollapsingTextHelper.java]: I thought that texture is only used on the older API and redundant since we target new API now. Let me know if this change was extra - I will restore the logic.

Comment (nickbutcher) [reply core/src/main/java/io/plaidapp/core/util/CollapsingTextHelper.java]: This is taken from the old design lib, I'd recommend keeping it in sync with any changes MDC have made.

https://github.com/material-components/material-components-android/blob/master/lib/java/com/google/android/material/internal/CollapsingTextHelper.java

Comment (emartynov) [reply core/src/main/java/io/plaidapp/core/util/CollapsingTextHelper.java]: Restored! Sorry!

Comment (emartynov) [reply core/src/main/java/io/plaidapp/core/util/CollapsingTextHelper.java]: The keeping it in sync I would like to do in the different changes.
